drivers/bus/pci/fdo.c

```diff
@@ -132,6 +154,22 @@ FdoEnumerateDevices(
 
                 Device->BusNumber = DeviceExtension->BusNumber;
 
+                if (PciIsDebuggingDevice(DeviceExtension->BusNumber, SlotNumber))
+                {
+                    Device->IsDebuggingDevice = TRUE;
+
+                    /*
+                     * ReactOS-specific: apply a hack
```

Comment:
Do you know what should be done ideally for removing the hack?

Reply:
Windows marks the device with the problem code CM_PROB_USED_BY_DEBUGGER. This is NT6.2+ stuff.

Reply:
Is it done by the kernel or by the driver?

Reply:
This is done by either the kernel or a user-mode component.